Clarify the differences between the SelectBox and Lookup widgets (#53)
DevExpress/devextreme-documentation: concepts/05 Widgets/Lookup/35 Lookup vs SelectBox.md

> The **Lookup** and [SelectBox](/api-reference/10%20UI%20Widgets/dxSelectBox '/Documentation/ApiReference/UI_Widgets/dxSelectBox/') widgets implement similar functionality - both of them consist of a text field and show a collection of items in the drop-down list, but they also have differences:

Reviewer: ```suggestion
The **Lookup** and [SelectBox](/api-reference/10%20UI%20Widgets/dxSelectBox '/Documentation/ApiReference/UI_Widgets/dxSelectBox/') widgets implement a similar functionality - while both consist of a text field and display a collection of items in the drop-down list, they also have differences:
```

- **Drop-down field content**   
The **SelectBox** has only a list of items in a drop-down field whereas **Lookup** also has a search field and the Cancel button.

- **Appearance on desktop and mobile devices**    

> The **SelectBox** looks the same on any platform while **Lookup** changes its appearance. The **Lookup**'s drop-down field is replaced with a popup window that occupies most of the screen. This design optimizes **Lookup** for mobile devices.

Reviewer: ```suggestion
The **SelectBox** looks the same on any platform, while **Lookup** changes its appearance. The **Lookup**'s drop-down field is replaced with a popup window that occupies most of the screen. This design optimizes **Lookup** for mobile devices.
```
